packages/usa-range/src/index.js

```diff
@@ -0,0 +1,52 @@
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+const behavior = require("../../uswds-core/src/js/utils/behavior");
+
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+
+const RANGE_CLASSNAME = `${PREFIX}-range`;
+const RANGE = `.${RANGE_CLASSNAME}`;
+
+/**
+ * Called when range slider has optional unit data attribute.
+ * Collects necessary attributes, then constructs and appends aria-valuetext attribute for screen reader context.
+ *
+ * @param {HTMLInputElement} targetRange - The range slider input element
+ */
+const updateCallout = (targetRange) => {
+  const rangeSlider = targetRange;
+  const unit = rangeSlider.getAttribute("data-optional-unit");
+  const val = rangeSlider.value;
+  const max = rangeSlider.getAttribute("max");
+  const callout = `${val} ${unit || ""} of ${max}`;
+
+  rangeSlider.setAttribute("aria-valuetext", callout);
+};
+
+/**
+ * Checks for optional unit data attribute. If attribute is present, updates aria-valuetext and adds input event listener.
```

Comment:
This comment feels like it describes `updateCallout`. Maybe it could more directly describe `enhanceSRContext`?

Reply:
Tbh the `enhanceSRContext` became a bit redundant. Simply calling `updateCallout` during initialization gets the job done as expected. I've removed `enhanceSRContext`. 

Let me know if you see any issue with that!

Resolved in [5d2afeb](https://github.com/uswds/uswds/pull/5472/commits/5d2afebc1f75c5b931e13619a15e123a098b0b1e)